objects.go

```diff
@@ -63,11 +97,130 @@ type Network struct {
 	Ea          EA     `json:"extattrs,omitempty"`
 }
 
+type Members []Member
+
+type ServiceStatus struct {
+	Desciption string `json:"description,omitempty"`
+	Service    string `json:"service,omitempty"`
+	Status     string `json:"status,omitempty"`
+}
+
+type LanHaPortSetting struct {
+	HaPortSetting  map[string]interface{} `json:"ha_port_setting,omitempty"`
+	LanPortSetting map[string]bool        `json:"lan_port_setting,omitempty"`
+}
+
+type NodeInfo struct {
+	HaStatus             string                 `json:"ha_status,omitempty"`
+	HwId                 string                 `json:"hwid,omitempty"`
+	HwModel              string                 `json:"hwmodel,omitempty"`
+	HwPlatform           string                 `json:"hwplatform,omitempty"`
+	HwType               string                 `json:"hwtype,omitempty"`
+	Lan2PhysicalSetting  map[string]bool        `json:"lan2_physical_setting,omitempty"`
+	LanHaPortSetting     LanHaPortSetting       `json:"lan_Ha_Port_Setting,omitempty"`
+	MgmtNetworkSetting   map[string]interface{} `json:"mgmt_network_setting,omitempty"`
```

Comment:
This has the same issue as above.

Various other fields have the same problem.

Reply:
i dont think its array. here's the sample

"mgmt_physical_setting": {
                    "auto_port_setting_enabled": true
                },

Reply:
Same here, should be a struct, i.e.:

`MgmtNetworkSetting     NetworkSettings`

where,

` type NetworkSettings struct {`
`       Address      string   //or however we store ipv4address`
`       Dscp                       uint`
`       Gateway                  string`

etc...

I said "array of struct" because I thought there's multiple of these.

Reply:
Sure, will fix it.